demo/app-root.ts

```diff
@@ -1,20 +1,98 @@
-import { html, css, LitElement } from 'lit';
-import { customElement } from 'lit/decorators.js';
-import '../src/your-webcomponent';
+/* eslint-disable import/no-duplicates */
+import { html, css, LitElement, SVGTemplateResult, svg } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import '../src/ia-dropdown';
```

Comment:
do we need to need to call `{ optionInterface }` explicitly in below line? I think you will get everything using this as well.

Reply:
yah, it's how ts works.. take a look [here](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html)